servicetest/cgroups/test_cgroups.py

```diff
@@ -0,0 +1,178 @@
+
+# Copyright (C) 2016 - 2017 Pelagicore AB
```

Comment:
I think the majority of files that already have this have used "2016-2017" so perhaps best to remove the spaces. I think it looks nicer if it's the same in all files.